docs/reference-guides/rancher-security/rancher-webhook-hardening.md

```diff
@@ -0,0 +1,133 @@
+---
+title: Hardening the Rancher Webhook
+---
+
+Rancher Webhook does not contain private information, but it can be vulnerable to DDoS attacks from inside the cluster. You can use network policies and authentication independently or in conjunction with each other to harden the webhook against attacks.
```

Comment:
```suggestion
Rancher Webhook is an important component within Rancher, playing a role in enforcing security requirements for Rancher and its workloads. To decrease its attack surface, access to it should be limited to the only valid caller it has: the Kubernetes API server. This can be done by using network policies and authentication independently or in conjunction with each other to harden the webhook against attacks.
```